### PR TITLE
Fix Ruby files

### PR DESCRIPTION
**Pull request by siaw23**
Completion of file fixing to match Ruby style guide. All tests pass.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
-gem 'rails', '~> 3.2.20'
 gem 'passenger'
+gem 'rails', '~> 3.2.20'
 
 gem 'rails-i18n', '~> 3.0.0'
 # run with `bundle install --without production` or `bundle install --without mysql` to exclude this
 group :mysql, :production do
-  gem 'mysql2', '~> 0.3.20'
+  # gem 'mysql2', '~> 0.3.20'
```
Comment (jywarren) [inline Gemfile]: Hi, was there a reason to disable this? Perhaps you didn't mean to check this one in?

**Comment by siaw23** [comment]
didn't mean to add that @jywarren fixed...

**Comment by siaw23** [comment]
@jywarren i guess this is OK now :)

**Comment by jywarren** [comment]
Thanks, I looked it over and hope to merge it in the next few days. Just
need to make time to rebase another change over it. Thank you!

**Comment by jywarren** [comment]
Wow, okay, i scanned through every line of this, and it was a lot. Very much appreciated, however I have to ask that any future changes of this scale be broken into smaller parts, because:

* I have to read the whole thing before merging
* I am reluctant to accept other PRs while this is unresolved, but it's so big, that puts a roadblock in the merge schedule
* I prefer we not change tests and non-test code at the same time

That said, my checks looked good and I'm very glad to accept this merge. Thank you so much!

**Pull request merged**